### PR TITLE
Add unit tests for ACS chat service

### PR DESCRIPTION
**Pull request by Leoaqr**

### CODE REVIEW

File: sdk/communication/AzureCommunicationChat/Tests/ChatClientTests.swift [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import XCTest
+import AzureCommunication
+import AzureCommunicationChat
+
+class ChatClientTests: XCTestCase {
```
Comment (knvsl) [inline sdk/communication/AzureCommunicationChat/Tests/ChatClientTests.swift:31]: This file should be removed - not part of unit tests right?

Comment (Leoaqr) [reply sdk/communication/AzureCommunicationChat/Tests/ChatClientTests.swift]: Yes, removed

**Comment by JoshuaLai** [comment]
It looks like the `swift build` command fails because it can't find the new OHHTTPStubs.
I think you just need to add this to the package file under dependencies by being more explicit. 
https://forums.swift.org/t/unable-to-import-multiple-targets-from-external-dependency/39030
Try something like this

**Pull request closed without merging**